sdk/cognitivelanguage/Azure.AI.Language.Text/samples/Sample13_AnalyzeTextOperationAsync_MultipleActions.md

```diff
@@ -0,0 +1,143 @@
+# Summarize documents with abstractive summarization
+
+This sample demonstrates how to perform multiple text analysis actions on one or more documents. These actions can include:
+
+- Named Entities Recognition
+- PII Entities Recognition
+- Linked Entity Recognition
+- Key Phrase Extraction
+- Sentiment Analysis
+- Extractive Summarization
+- Custom Named Entity Recognition
+- Custom Text Classification
+
+## Create a `TextAnalysisClient`
+
+To create a new `TextAnalysisClient`, you will need the service endpoint and credentials of your Language resource. To authenticate, you can use the [`DefaultAzureCredential`][DefaultAzureCredential], which combines credentials commonly used to authenticate when deployed on Azure, with credentials used to authenticate in a development environment. In this sample, however, you will use an `AzureKeyCredential`, which you can create with an API key.
+
+```C# Snippet:CreateTextClient
+Uri endpoint = new Uri("<your endpoint>");
+AzureKeyCredential credential = new("your apikey");
```

Comment:
Do we want to think about adding docs on how to call these endpoints with:

1keyv2
managed identity

etc (basically - all the options that are non-key based since we are recommending no more keys)

Reply:
Nvm on this one - Bidisha pointed me at this - https://github.com/Azure/azure-sdk-for-net/tree/07243563fcdf1aa9b19f88e6b2952fb111d232ba/sdk/cognitivelanguage/Azure.AI.Language.Conversations#create-a-client-using-azure-active-directory-authentication